### DISCUSSION TITLE
Sort by recency - Enable sorting by `uploaded_at` for files and requests

### DISCUSSION THREAD

**Discussion by RM-FM** [AzuraCast/AzuraCast#7331] [Ideas]
It's great to see that `uploaded_at` is now available as native file metadata. It would be fantastic to include this in the `sortQueryBuilder` for the `/files` and `/requests` API endpoints, as it would allow us to search and sort by the most recent songs.

This feature could also be valuable for public pages.

Please also see: #7196

**Comment by BusterNeece** [comment]
@RM-FM I saw your post on the other item, but this isn't the channel we use for making feature requests: you should submit it over at https://features.azuracast.com/ so we can catalog it appropriately.

**Reply by RM-FM** [reply]
@BusterNeece Thanks for the hint! Missed that...
https://features.azuracast.com/suggestions/568040/sort-by-recency-enable-sorting-by-uploaded_at-for-files-and-requests
